### ISSUE TITLE
inactivity_timeout for the vault endpoint isn't configurable/too short

### ISSUE THREAD

**Issue by steveberryman** [criteo/consul-templaterb#76]
We run a somewhat unconventional consul/vault setup, in that our clusters are far away from some of our servers and thus latency is pretty poor. With the hard-coded `inactivity_timeout` for vault of 1s here

https://github.com/criteo/consul-templaterb/blob/fa13ad2143726dad0fbdc32e0e4c15affb7c370d/lib/consul/async/vault_endpoint.rb#L234

consul-templaterb often times out during a run for some keys. I confirmed that increasing this to a longer time fixes the issue. Potentially somewhat of a niche request, but making this configurable would allow increase the timeout in cases where vault was a stupid distance away from the host consul-templaterb was running on.

**Comment by pierresouchay** [comment]
@steveberryman Indeed, this is pretty hardcoded... there are lots of places where we borrow parameters from the command line, would you be interested with providing a patch?

**Comment by steveberryman** [comment]
Yeah I’m sure I could do that :)
